### PR TITLE
feat: add feature flag constant to the UI contants

### LINKED ISSUE #357: Add feature flag for MVP-search
In order to do development on the new user interface, we need a feature flag for MVP-search.

### PR DESCRIPTION
**Pull request by qcdyx**
**Summary:**

Closes #357 

**Expected behavior:** 

1. enableMVPSearch feature flag added in all environments on Firebase 
2. documentation added in the doc folder of the repo

image

**Testing tips:**

Provide tips, procedures and sample files on how to test the feature.
Testers are invited to follow the tips AND to try anything they deem relevant outside the bounds of the testing tips. 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [x] Add or update any needed documentation to the repo
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: docs/feature-flag/feature_flag [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# How to Add a Feature Flag on Firebase
```
Comment (davidgamez) [inline docs/feature-flag/feature_flag:1]: Add the _md_ file extension so GH can render it correctly.

File: docs/feature-flag/feature_flag [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## 1. Go to Remote Config on Firebase
```
Comment (davidgamez) [inline docs/feature-flag/feature_flag:3]: [suggestion]: add the screenshots to each step

File: docs/feature-flag/feature_flag (hunk 1 above)
Comment (qcdyx) [reply docs/feature-flag/feature_flag]: Done.

File: docs/feature-flag/feature_flag (hunk 2 above)
Comment (qcdyx) [reply docs/feature-flag/feature_flag]: Added.

**Review by davidgamez — APPROVED** [review]
LGTM!

**Pull request merged**